### DISCUSSION TITLE
Noice does not autocomplete while typing (cmdline, after hitting `:`)

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [LazyVim/LazyVim#2457] [Q&A] [answered]
I just saw this YouTube video where there is autocompletion of the cmdline using Noice: https://youtu.be/upM6FOtdLeU?t=157

What's odd here is I don't see this completion while typing. Is this meant to work out of the box with LazyVim?
I'm wondering if I have some interfering settings somewhere (not sure what that might be) which prevents this completion from showing...

**Comment by dpetka2001** [comment, marked as answer]
I believe this comes from [cmp-cmdline](https://github.com/hrsh7th/cmp-cmdline) which is not included in LazyVim by default. You can see #2072 for an example.

**Reply by fredrikaverpil** [reply]
Holy moly, you made my day. Such a quality of life improvement! ⭐ 

Thank you 🙌 @dpetka2001 😄
